src/commands/forum.ts

```diff
@@ -15,10 +15,12 @@ export const forum: CommandDef = {
       const forumEmbed: MessageEmbed = new MessageEmbed()
         .setTitle('Latest Forum Activity')
         .setDescription('Here are the five most recent posts.');
-      result.forEach((el: Record<string, unknown>) =>
+      result.forEach((el: Topic) =>
```

Comment:
The `result` variable can be renamed to something like `topics` and typed on the `.slice(0, 5)`  with the type `Topic[]`.